dftparse/pwscf/stdout_parser.py

```diff
@@ -1,173 +1,481 @@
-  (lambda x: "total   stress" in x, _parse_stress),
-  (lambda x: "Forces acting on atoms" in x, _parse_forces),
-  _gen_energy_contrib("one-electron"),
-  _gen_energy_contrib("hartree"),
-  _gen_energy_contrib("xc"),
-  _gen_energy_contrib("ewald")
+    (lambda x: 'Program PWSCF' in x, _parse_header),
+    (lambda x: 'Reading input from' in x, _parse_input_filename),
+    (lambda x: 'PseudoPot. #' in x, _parse_pseudopotential),
+    (lambda x: 'bravais-lattice index' in x, _parse_bravais_lattice),
+    (lambda x: 'lattice parameter' in x, _parse_lattice_parameter),
+    (lambda x: 'crystal axes:' in x, _parse_cell_vectors),
+    (lambda x: 'CELL_PARAMETERS ' in x, _parse_cell_vectors),
+    (lambda x: 'unit-cell volume' in x, _parse_unit_cell_volume),
+    (lambda x: 'number of atoms/cell' in x, _parse_n_atoms_per_cell),
+    (lambda x: 'number of atomic types' in x, _parse_n_atom_types),
+    (lambda x: 'number of electrons' in x, _parse_n_electrons),
+    (lambda x: 'kinetic-energy cutoff' in x, _parse_kinetic_energy_cutoff),
+    (lambda x: 'charge density cutoff' in x, _parse_charge_density_cutoff),
+    (lambda x: 'mixing beta ' in x, _parse_mixing_beta),
+    (lambda x: 'convergence threshold ' in x, _parse_scf_conv_threshold),
+    (lambda x: 'convergence thresholds ' in x, _parse_ionic_conv_threshold),
+    (lambda x: 'criteria: energy ' in x, _parse_ionic_conv_threshold),
+    (lambda x: 'Exchange-correlation' in x, _parse_xc),
+    (lambda x: 'number of k points=' in x, _parse_kpoints_block),
+    (lambda x: 'Fermi energy is' in x, _parse_fermi_energy),
+    (lambda x: '!    total energy' in x, _parse_total_energy),
+    _gen_energy_contrib('one-electron'),
+    _gen_energy_contrib('hartree'),
+    _gen_energy_contrib('xc'),
+    _gen_energy_contrib('ewald'),
+    _gen_energy_contrib('smearing'),
+    (lambda x: 'Hubbard energy' in x, _parse_hubbard_energy),
+    (lambda x: 'Forces acting on atoms' in x, _parse_forces),
+    (lambda x: 'total   stress' in x, _parse_stress_and_pressure),
+    (lambda x: 'Simplified LDA+U calculation' in x, _parse_ldau_parameters),
+    (lambda x: 'bfgs converged in' in x, _parse_n_bfgs_steps),
+    (lambda x: 'convergence has been ' in x, _parse_n_steps_for_sc),
+    (lambda x: 'PWSCF' and 'WALL' in x, _parse_total_cpu_time),
+    (lambda x: 'atom                  pos' in x, _parse_atomic_positions),
+    (lambda x: 'ATOMIC_POSITIONS' in x, _parse_atomic_positions),
+    (lambda x: 'Starting magnetic ' in x, _parse_starting_mag_structure),
+    (lambda x: 'total magnetization' in x, _parse_total_magnetization),
+    (lambda x: 'absolute magnetization' in x, _parse_absolute_magnetization),
+    (lambda x: 'Magnetic moment per site' in x, _parse_site_proj_quantities),
+    (lambda x: 'warning' in x.lower(), _parse_warning),
 ]
 
+
 class PwscfStdOutputParser(BlockParser):
 
     def __init__(self, rules=base_rules):
-        BlockParser.__init__(self)
+        super(PwscfStdOutputParser, self).__init__()
```

Comment:
Why this change?  They look equivalent to me, and the way it was is more explicit IMO.